### PR TITLE
Add Mastodon button using NickKaramoff's toot.karamoff.dev

### PR DESCRIPTION
**Pull request by andybalaam**
Adding a Mastodon share button.

This is a much simpler version of #23 now that @NickKaramoff magicked up https://github.com/NickKaramoff/toot

### CODE REVIEW

**Review by NickKaramoff — APPROVED** [review]
Wonderful! I'll test it on my side and get the release ready

File: src/networks.js [hunk 1]
```diff
@@ -38,6 +38,11 @@ const NETWORKS = {
     iconWhenText: 'data:image/svg+xml,%3Csvg fill="%23fff" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"%3E%3Cpath d="M20.447 20.452h-3.554v-5.569c0-1.328-.027-3.037-1.852-3.037-1.853 0-2.136 1.445-2.136 2.939v5.667H9.351V9h3.414v1.561h.046c.477-.9 1.637-1.85 3.37-1.85 3.601 0 4.267 2.37 4.267 5.455v6.286zM5.337 7.433a2.062 2.062 0 01-2.063-2.065 2.064 2.064 0 112.063 2.065zm1.782 13.019H3.555V9h3.564v11.452zM22.225 0H1.771C.792 0 0 .774 0 1.729v20.542C0 23.227.792 24 1.771 24h20.451C23.2 24 24 23.227 24 22.271V1.729C24 .774 23.2 0 22.222 0h.003z"/%3E%3C/svg%3E',
     url: (d) => `https://www.linkedin.com/shareArticle?mini=true&url=${d.url}&title=${d.title}`,
   },
+  mastodon: {
+    color: '#1f232b',
```
Comment (NickKaramoff) [inline src/networks.js:42]: One change only — I would change the button colour to Mastodon's blue.

```suggestion
    color: '#3088D4',
```

According to the Mastodon [Press Kit](https://joinmastodon.org/press-kit.zip), it is the intended background to use with a white monochrome icon:

Excerpt from Mastodon Press Kit

**Review by NickKaramoff — CHANGES_REQUESTED** [review]
Only one small change before I'll ship it — the colour of the button

File: src/networks.js (hunk 1 above)
Comment (andybalaam) [reply src/networks.js:42]: Yep, that colour was from when I had the non-white logo.  Another nice catch.

**Review by NickKaramoff — APPROVED** [review]

**Comment by NickKaramoff** [comment]
Shipped with [v1.5.0](https://github.com/NickKaramoff/shareon/releases/tag/v1.5.0) :tada:

Thank you for your contribution!

**Comment by NickKaramoff** [comment]
Note that it will take some time for JSDelivr to invalidate its caches and ship the changes in `@1` version of the URL 😅

**Comment by andybalaam** [comment]
Awesome, thank you!

**Pull request merged**